### DISCUSSION TITLE
Chrome Version missmatch

### DISCUSSION THREAD

**Discussion by ds2edv** [chocolatey-community/chocolatey-packages#2455] [General]
Still wrong versions. The package is 124 but the downloaded MSI is 123. Looks like the Enterprise MSI is extended stable since this year. Check the Downloadpage for the Enterprise MSI it also shows a 123 as the actual Version. You have to grap the Version number from the extended chanel and not the stable chanel from the API. But Looks like Google hast issues here - all chanels that are not the stable stopped publishing new versions since version 122.

**Comment by pauby** [comment]
The version on the [Chocolatey Community Repository is 124.0.6367.60](https://community.chocolatey.org/packages/googlechrome#versionhistory) and the version installed, currently is 124.0.6367.61. See my testing below.

![image](https://github.com/chocolatey-community/chocolatey-packages/assets/12760779/6e339c2e-4b95-4025-8282-9d5383d89d68)

Note that the checksum is wrong as the version of the package is for the 124.0.6367.60 version of the installer, and with the later version of that installer available the checksums no longer match (Google Chrome only has one installer, no matter the version). That will be updated shortly when the automation to update the package, runs again.

There is **no way** to install an older version of Google Chrome using the Chocolatey package using the free and open-source version of Chocolatey CLI. You may be able to do that with the licensed editions (which cache the installers making it possible to go back to previous versions).
